packages/p2p/src/components/online-status/online-status-avatar.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
-import { Text } from '@deriv/components';
-import PropTypes from 'prop-types';
 import { observer } from 'mobx-react-lite';
+import { Text } from '@deriv/components';
 import { generateHexColourFromNickname, getShortNickname } from 'Utils/string';
 import OnlineStatusIcon from './online-status-icon';
-import './online-status-avatar.scss';
 
-const OnlineStatusAvatar = ({ is_online, nickname, size, text_size }) => {
```

Comment:
dont we need to put this in separate folder with its own `index.ts`?

Reply:
not needed, since its a subpart of the main component online-status. had discussed with farrah